sample_code_submission/README.md

```diff
@@ -0,0 +1,8 @@
+# Sample code submission
+This folder contains the sample code submission
+## Description
+The main part of the sample code submission is in the `model.py` which contains the `Model` class. 
+* The model class uses the BDT classifier using [xgboost](https://xgboost.ai/) defined in [boosted_decision_tree](/sample_code_submission/boosted_decision_tree.py). 
+* The statistical analysis part is written in [statistical_analysis](/sample_code_submission/statistical_analysis.py) 
+
+Workings of the Statistical Analysis part can be found in our [whitepaper](https://example.com/whitepaper)
```

Comment:
you can put a pdf link where we can put the white paper
https://fair-universe.lbl.gov/whitepaper.pdf